utils/__mocks__/ifPortIsFreeDo.js

```diff
@@ -0,0 +1,3 @@
+const mock = jest.fn((port, todo) => { todo(); });
```

Comment:
By Jest's convention a mock lives adjacent to the module it is mocking, with the exception of a `node_modules` mock which would live at the top-level `__mocks__` directory.

https://facebook.github.io/jest/docs/manual-mocks.html#content